odoo/api.py

```diff
@@ -806,6 +806,11 @@ def get(key, get_context=self.context.get):
             self._cache_key[field] = result
             return result
 
+    def profile(self, **kwargs):
```

Comment:
What is the added value of
```python
with self.env.profile(...):
```
compared to
```python
with odoo.tools.profiler.Profiler(...):
```

Reply:
ease of use (no import needed, quick access) but nothing else. I can remove it.